pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.mycompany</groupId>
+    <groupId>cz.muni.fi.pa165.pokemoni</groupId>
```

Comment:
*pokemon
lets keep everything in english, shall we:)

Reply:
I thought our group is called 'pokemoni' so that was what i put there.
The structure of package name should consist of organization's (teams') place from the point of the whole world. But whatever, i can change it to pokemons (or pokemon), depends on what you prefer.

Reply:
It is up to you , your choice